Src/Support/Google.Apis.Auth/OAuth2/Requests/GoogleWifStsTokenRequest.cs

```diff
@@ -0,0 +1,81 @@
+﻿/*
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Util;
+
+namespace Google.Apis.Auth.OAuth2.Requests
+{
+    /// <summary>
+    /// OAuth 2.0 subject token exchange request as defined in
+    /// https://datatracker.ietf.org/doc/html/rfc8693#section-2.1.
+    /// This is a partial definition of the spec as required to support Google WIF.
+    /// </summary>
+    internal class GoogleWifStsTokenRequest
+    {
+        /// <summary>
+        /// Gets the grant type for this request.
+        /// Only <code>urn:ietf:params:oauth:grant-type:token-exchange</code> currently supported.
```

Comment:
Add "is" after the code element? (And presumably it wouldn't really hurt to include a setter, so long as we initialize it with the right value? Ditto for RequestedTokenType.)